config/application.rb

```diff
@@ -74,5 +74,7 @@ class Application < Rails::Application
 
     # Default headers
     config.action_dispatch.default_headers["X-XSS-Protection"] = "0"
+
+    config.host_authorization = {exclude: ->(request) { request.path =~ /health_check/ }}
```

Comment:
Do we need to include the `/health-check/` path as well?

Reply:
I was counting on the fact that the app responds to both health-check and health_check requests, but we might, not sure.

Reply:
I think you're right, the request needs to match, even if it's processed by the same controller. PR updated.